### PR TITLE
[rtsan][NFC] Add documentation link to Function Effects

### PR DESCRIPTION
**Pull request by cjappl**

### CODE REVIEW

**Comment by cjappl** [comment]
CC for review @dougsonos

File: clang/docs/RealtimeSanitizer.rst [hunk 1]
```diff
@@ -11,11 +11,16 @@ RealtimeSanitizer (a.k.a. RTSan) is a real-time safety testing tool for C and C+
 projects. RTSan can be used to detect real-time violations, i.e. calls to methods
 that are not safe for use in functions with deterministic run time requirements.
 RTSan considers any function marked with the ``[[clang::nonblocking]]`` attribute
-to be a real-time function. If RTSan detects a call to ``malloc``, ``free``,
-``pthread_mutex_lock``, or anything else that could have a non-deterministic
-execution time in a function marked ``[[clang::nonblocking]]``
+to be a real-time function. At run-time, if RTSan detects a call to ``malloc``, 
+``free``, ``pthread_mutex_lock``, or anything else that could have a 
+non-deterministic execution time in a function marked ``[[clang::nonblocking]]``
```
Comment (fmayer) [inline clang/docs/RealtimeSanitizer.rst:16]: > or anything else that could have a non-deterministic execution time

that sounds like a guarantee. Isn't it more things that we _know_ to have a non-deterministic runtime

**Review by Sirraide — COMMENTED** [review]
Effects paragraph lgtm. I’m not really familiar w/ rtsan, so I can’t comment on the rest.

File: clang/docs/RealtimeSanitizer.rst (hunk 1 above)
Comment (cjappl) [reply clang/docs/RealtimeSanitizer.rst:16]: I agree, let me fix this in a future PR if that is alright - I want to keep this one focused.

The only non-whitespace change in this paragraph is the addition of "At run-time" the rest is git having problems with the new line breaks

**Review by fmayer — APPROVED** [review]

**Comment by dougsonos** [comment]
LGTM

**Pull request merged**